lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/collection/includeExcludeWidget.py

```diff
@@ -163,17 +310,19 @@ def removeItemToCollection(self, item):
                 self.control._collection.GetExcludesRel().RemoveTarget(item)
 
         self.control.update()
-    '''
+    """
 
     def _validatePath(self, collection, path):
-       
+
         if not Sdf.Path.IsValidPathString(path):
             raise ValueError("Invalid sdf path: " + path)
 
         stage = self.control._collection.GetPrim().GetStage()
         prim = stage.GetPrimAtPath(Sdf.Path(path))
 
         if not prim or not prim.IsValid():
-            raise ValueError("Value must be a float or an int")("Error: The dragged object is not in the same stage as the collection. Ensure that objects belong to the same stage before adding them")
-        
-        return True
\ No newline at end of file
+            raise ValueError("Value must be a float or an int")(
+                "Error: The dragged object is not in the same stage as the collection. Ensure that objects belong to the same stage before adding them"
+            )
```

Comment:
I saw this code and wondered if this was code generated by AI?

In any case, error will have to be reported by going through the data-model class to support DCC-specific error reporting mechanisms.